module_cheatsheets/Intro_to_R_Tidyverse_cheatsheet.md

```diff
@@ -0,0 +1,82 @@
+| Base `R`          | [`rm(x)`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/remove)              | Remove                | Removes object(s) `x` from your environment                                                                                                                                                         |
+| Base `R`          | [`==, <=, >=, !=`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/Comparison.html)       | Relational Operators  | These are binary operators which allow for the comparison of values in an object.                                                                                                                 |
+| Base `R`          | [`str(x)`](https://www.rdocumentation.org/packages/utils/versions/3.5.1/topics/str)               | Object Structure      | Gets a summary of the object `x` structure.                                                                                                                                                         |
+| Base `R`          | [`head(x)`](https://www.rdocumentation.org/packages/utils/versions/3.5.1/topics/head)             | Head                  | Returns the top 6 rows of an object in the environment by default. You can specify how many rows you want by including the `n = `argument.                                                        |
+| Base `R`          | [`tail(x)`](https://www.rdocumentation.org/packages/utils/versions/3.5.1/topics/head)             | Tail                  | Returns the bottom 6 rows of an object in the environment. You can specify how many rows you want by including the `n =` argument.                                                                  |
+| Base `R`          | [`as.factor(x)`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/factor)       | As Factor             | Coerces object x into a factor (which is used to represent categorical data). This function can be used to coerce object `x` into other data types i.e.. as.character, as.data.frame, as.matrix, etc. |
+| Base `R`          | [`dim(x)`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/dim)                | Dimensions summary    | Returns the dimensions of object `x`.                                                                                                                                                               |
+| Base `R`          | [`data.frame()`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/data.frame)   | Data Frame            | Creates a data.frame where the named arguments will be the same length                                                                                             |
+| Base `R`          | `[[x]]`[click here](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/Extract)             |  Double bracket       | Extracts `x` from a nested list                                |
```

Comment:
```suggestion
| Base `R`          | [`[[x]]`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/Extract)             |  Double bracket       | Extracts `x` from a nested list                                |
```

Reply:
I tried this format and it disrupted the format of the table (the information from the following two columns becomes squashed into this column). Any thoughts on that?

Reply:
Don't know _why_ but I am pretty sure that a switch to HTML format will do it:
```suggestion
| Base `R`          | <a href = "https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/Extract">`[[x]]`</a>             |  Double bracket       | Extracts `x` from a nested list                                |
```

Reply:
Great, that works